autoware_cmake/cmake/autoware_package.cmake

```diff
@@ -44,6 +44,12 @@ macro(autoware_package)
     add_compile_definitions(ROS_DISTRO_HUMBLE)
   endif()
 
+  # Set common system includes
```

Comment:
I think this is not the right place for this, this means that every package will call `find_package` even if they don't need Eigen. I'd update the packages that actually need Eigen to call `include_directories(SYSTEM...) `

Reply:
I can remove this if https://github.com/ament/ament_cmake/pull/385 is released.

Reply:
I think we can manually call `include_directories(SYSTEM ...)` in those packages that depend on Eigen, even if we are using `ament_cmake_auto`. I'm going to test this.

Reply:
The [interpolation package](https://github.com/autowarefoundation/autoware.universe/tree/main/common/interpolation)
 has an error, but it doesn't depend on Eigen.

Reply:
Even more reason for not calling `find_package(Eigen ...)` if the interpolation package does not depend on Eigen, adding that to `autoware_package` would be too much, every package would include the Eigen headers no matter what.